src/intl.ts

```diff
@@ -403,6 +403,14 @@ export const buttonMessages = defineMessages({
     id: "rbrahO",
     defaultMessage: "Close",
   },
+  cancelOrder: {
+    id: "8fI5PB",
+    defaultMessage: "Cancel order",
+  },
+  keepOrder: {
+    id: "CswOOG",
+    defaultMessage: "Keep order",
+  },
```

Comment:
`src/intl.ts` is a file containing very generic messages, the ones you added refer to the order cancel dialog specifically, so I think they should be put either:
- directly inside `src/orders/components/OrderCancelDialog/OrderCancelDialog.tsx`
- inside new dedicated messages.ts file with strings for this component. You can see example how it's done in `src/orders/components/OrderChangeWarehouseDialog/messages.ts`